Update cd-integrations-supported.md (#4719)
harness/developer-hub: docs/continuous-delivery/shared/cd-integrations-supported.md

    - 1.16
    - 1.27
    - We support what each of the Cloud Providers support. We recommend users to keep their binary versions up to date.
    - By default, Harness ships with kubectl client - 1.24.3

>     - Harness has certified - 1.25, 1.26, 1.27 versions of kubectl, users will need to install the respective client version the delegate for Harness to leverage it.

Reviewer: ```suggestion
    - Harness has certified versions 1.25, 1.26, and 1.27 of kubectl. You must users install the respective client version of the delegate for Harness to leverage it.
```

>     - Harness has certified versions 1.25, 1.26, and 1.27 of kubectl. You must users install the respective client version of the delegate for Harness to leverage it.

Reviewer: ```suggestion
    - Harness has certified versions 1.25, 1.26, and 1.27 of kubectl. You must install the respective client version of the delegate for Harness to leverage it.
```
